app/components/agenda-manager/agenda-table/row.hbs

```diff
@@ -1,11 +1,6 @@
 <DraggableObject @tagName="tr" @content={{@item}} @isSortable={{true}}>
-  <td>
-    <AuButton @skin="tertiary">
```

Comment:
Button seemed unnecessary since the whole tr is draggable. It also added unwanted styles to the icon, which is why the inline styles were needed in the first place.

Reply:
I think the button was added for accessibility reasons, but not 100% sure.

Reply:
Yea, that makes sense. I don't think only adding the button will make it accessible though. It doesn't have any keyboard navigation as far as I can tell and no readable instructions either. But I'll add it back anyway.

Reply:
Reverted to the previous version and added the cursor override.